### PR TITLE
Files Updated Succesfully

### PR DESCRIPTION
**Pull request by Soumyajit2825**
@Axe-Pearl Done some small changes. I'm doing the PR. Hope you will find it helpful. Thank You 🌹

### CODE REVIEW

File: game.html [hunk 1]
```diff
@@ -4,8 +4,17 @@
     <meta charset="utf-8">
     <title>Tic Tac Toe</title>
     <link rel="stylesheet" href="game.css">
+    <style>
```
Comment (Axe-Pearl) [inline game.html:7]: Please remove this style

File: game.html [hunk 2]
```diff
@@ -4,8 +4,17 @@
...
+      body  {
+        background-image: url("paper.gif");
+        background-color: #cccccc;
+      }
+      </style>
   </head>
   <body>
+    <div id="main">
+      <p>Game starts by just Tap on box<br><br>First Player starts as<b>Player X</b><br>And<br>Second Player as <b>Player 0</b></p>
```
Comment (Axe-Pearl) [inline game.html:16]: You can remove the following instruction, we don't need them they are just decreasing UI quality

Comment (Axe-Pearl) [inline game.html:16]: Remove this Paragraph tag

**Review by Axe-Pearl — COMMENTED** [review]
@Soumyajit2825 Please do check the improvement you have to make in your pull request, then I will merge it

**Comment by Soumyajit2825** [comment]
sure

**Pull request merged**